.github/workflows/pr-test-lint.yml

```diff
@@ -135,13 +135,19 @@ jobs:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
           filterOutClosed: true
           filterOutDraft: true
-      - name: Approve and merge a PR
-        run: |
-          gh pr review ${{ steps.pr.outputs.number }} --approve
```

Comment:
Sorry, I missed this one on the first review:
We need to approve the PR. Or is one of the actions taking care of this?